packages/playwright-core/src/server/launchApp.ts

```diff
@@ -46,9 +46,9 @@ export async function launchApp(browserType: BrowserType, options: {
     channel: !options.persistentContextOptions?.executablePath ? findChromiumChannel(options.sdkLanguage) : undefined,
     noDefaultViewport: true,
     ignoreDefaultArgs: ['--enable-automation'],
-    colorScheme: 'no-override',
     acceptDownloads: isUnderTest() ? 'accept' : 'internal-browser-default',
     ...options?.persistentContextOptions,
+    colorScheme: options?.persistentContextOptions?.colorScheme ?? 'no-override',
```

Comment:
Should we do the same for other options present here and in `persistentContextOptions`?